api/user.proto

```diff
@@ -175,6 +175,8 @@ message UserSession {
   google.protobuf.Timestamp created_at = 3 [(gogoproto.nullable) = false, (gogoproto.stdtime) = true];
   google.protobuf.Timestamp updated_at = 4 [(gogoproto.nullable) = false, (gogoproto.stdtime) = true];
   google.protobuf.Timestamp expires_at = 5 [(gogoproto.stdtime) = true];
+  // The session secret is used to compose an authorization key and is never returned.
+  string session_secret = 7;
```

Comment:
Why not `6`?

Also, is there no validation here at all? Do we ever receive this value from a client?

Reply:
Indeed. Leftover from when I added a token id as well.